source/backend/dal/Repositories/ManagementFileRepository.cs

```diff
@@ -460,10 +467,16 @@ private IQueryable<PimsManagementFile> GetCommonManagementFileQueryDeep(Manageme
                 predicate = predicate.And(disp => disp.PimsManagementFileTeams.Any(x => x.OrganizationId == filter.TeamMemberOrganizationId.Value));
             }
 
+            if (filter.ManagementRegionCode.HasValue)
+            {
+                predicate = predicate.And(disp => disp.RegionCode == filter.ManagementRegionCode);
+            }
+
```

Comment:
This section and the one in line 422 seem to do the same. What is the difference between `filter.RegionCode` and `filter.ManagementRegionCode`? They both seem to be doing the same in this query